index.html

```diff
@@ -240,13 +250,32 @@ <h2>Contact me</h2>
           needs coding.
           I’d love to help with it!</p>
       </header>
-      <form action="submit">
-        <input type="text" placeholder="Lorem Ipsum">
-        <input type="email" placeholder="loremipsum@gmail.com">
-        <textarea name="texto" rows="4" cols="30">Write your message here...</textarea>
-        <input class="btn" type="submit" value="Get in touch">
+
+      <form action="https://formspree.io/f/mbjezeel" method="post">
+        <ul>
+          <li>
+
+            <input type="text" id="name" name="user_name" placeholder="Lorem Ipsum">
+          </li>
+
+          <li>
+
+            <input type="email" id="mail" name="user_email" placeholder="loremipsum@gmail.com" required>
+          </li>
+
+          <li>
+
+            <textarea id="msg" name="user_message" rows="4" cols="30" required>Write your message here...</textarea>
```

Comment:
- [ ] As mentioned by the previous reviewer and for the project requirement, kindly make sure your `Message` `input` has `maxlength="500"` attribute.

- [ ] To match the your chosen design and as mentioned by the previous reviewer, I think it would be better if you use the `Write your message here...` text in a `placeholder` attribute instead of being the value of the `textarea` tag.
```HTML
<textarea id="msg" .... maxlength="500" placeholder="Write your message here..."></textarea>
```

Reply:
@tresorsawasawa  Thanks for the border-radius tip!